exercises/Anthony_Lim/ex02_oo_basics/src/shape_sorter.cpp

```diff
@@ -0,0 +1,85 @@
+//This is the introduction exercise 2. The work has been completed by Anthony Lim 
+/*
+------------------------------------------------------------------------------
+Include statements:
+
+ 	internal headers - same project
+ 	internal headers - different project
+ 	3rd part headers
+	system
+-------------------------------------------------------------------------------
+*/
+
+#include "shape_sorter.h" 
+#include<string> 
+#include<algorithm> 
+/*
+Aurthor: Anthony Lim
+data: 10/01/2017
+*/
+
+/*
+This body file is for the shape_sorter class 
+*/
+void ShapeSorter::PrintMatchedShape(std::string this_shape) const 
+{
+	for(int j =0;j<m_shapes.size();j++)
```

Comment:
Consider using a range-based for-loop. This is now standardly used in the Mantid code base. Best to use it here too.